application/models/services/QuestionGroupService.php

```diff
@@ -251,7 +264,10 @@ public function importQuestionGroup(int $surveyId, string $tmpDir, string $trans
             || $_FILES['the_file']['error'] == 2
         ) {
             $fatalerror = sprintf(
-                gT("Sorry, this file is too large. Only files up to %01.2f MB are allowed."),
+                gT(
+                    'Sorry, this file is too large. '
+                    . 'Only files up to %01.2f MB are allowed.'
+                ),
```

Comment:
Doesn't that break our translation functionality?

Reply:
No. The concatenated string is the same.

Reply:
Hm what about the script that scans the codebase looking for strings inside gT()?